content/docs/guides/configuration.md

```diff
@@ -42,7 +42,7 @@ by changing the `defaultSchema` property to some other value. For example, to ch
 
 ### Running operations in context
```

Comment:
we should probably move this to the docs which are specifically about `Table` and/or `Operation` now